dev/gpctl/cmd/api/workspaces.go

```diff
@@ -68,6 +69,58 @@ func newWorkspacesGetCommand() *cobra.Command {
 	return cmd
 }
 
+func newWorkspacesOwnerTokenCommand() *cobra.Command {
```

Comment:
style nit: we are usually following the cobra convention for commands, i.e. each command gets their own file (see the other `gpctl` commands). What's the motivation for moving to a different style here?

Reply:
No motivation really 😄 . I was just looking to get something working as I learn how to work on the public api. I'll move it before taking the PR out of draft.

Reply:
The original reason was that we only had a single command so it didn't make sense to spread it over too many files. 
Personally, I think this should be structured in packages rather than files which are deeply nesting but that's a different topic entirely.